superset-frontend/src/dashboard/types.ts

```diff
@@ -268,4 +268,5 @@ export enum MenuKeys {
   ToggleFullscreen = 'toggle_fullscreen',
   ManageEmbedded = 'manage_embedded',
   ManageEmailReports = 'manage_email_reports',
+  ViewTableInfo = 'view_dataset_info',
```

Comment:
can we make the naming consistent? `ViewDatasetInfo`.

also the component name `ViewTableInfoModal` below

Reply:
Switched the naming to table info instead of dataset info